ansible/configs/satellite-vm/default_vars_ocp.yml

```diff
@@ -0,0 +1,100 @@
+################################################################################
+### OSP Environment variables
+################################################################################
+
+
+cloud_provider: osp                               
+install_student_user: false
+
+
+ansible_user: cloud-user
+remote_user: cloud-user
+osp_cluster_dns_zone: red.osp.opentlc.com
+osp_cluster_dns_server: ddns01.opentlc.com
+use_dynamic_dns: true
+osp_project_create: true
+student_name: student
+admin_user: opentlc-mgr
+
+
+bastion_instance_type: 2c2g30d
+satellite_instance_type: 8c32g100d
+
+___image: rhel-server-7.7-update-2 
```

Comment:
why so many _____ for this var? Makes it hard to determine if there is a mistake at a glance.

Reply:
changed to one _  thanks.